manifests/00-crd-extension-console-notification.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: consolenotifications.console.openshift.io
+  annotations:
+    displayName: ConsoleNotification
+    description: Extension for configuring openshift web console notifications.
+spec:
+  scope: Cluster
+  group: console.openshift.io
+  versions:
+  - name: v1
+    served: true
+    storage: true
+  names:
+    plural: consolenotifications
+    singular: consolenotification
+    kind: ConsoleNotification
+    listKind: ConsoleNotificationList
+  additionalPrinterColumns:
+  - name: Text
+    type: string
+    JSONPath: .spec.text
+  - name: Location
+    type: string
+    JSONPath: .spec.location
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          description: Represents console notification spec
+          required:
+          - text
+          - location
+          properties:
+            text:
+              type: string
+              description: Text of the notification
+            location:
+              type: string
+              description: The location of the notification (BannerTop, BannerBottom, BannerTopBottom)
+              pattern: '\bBannerTop\b|\bBannerBottom\b|\bBannerTopBottom\b'
```

If this is a substring match, so we'd need something like

```suggestion
              pattern: '^(BannerTop|BannerBottom|BannerTopBottom)$'
```

---

Updating now.